virtualenv.py

```diff
@@ -1333,9 +1333,12 @@ def copy_required_files(src_dir, lib_dir, symlink):
 
 def copy_license(prefix, dst_prefix, lib_dir, symlink):
     """Copy the license file so `license()` builtin works"""
+    lib64_dir = lib_dir.replace("lib", "lib64")
```

Comment:
hmmm, shouldn't this split and then join so it is only removing a segment named `lib` -- this'll potentially fail for `/foolib/lib/python3.6` or some such

Reply:
probably right